src/components/SidePageComponent.js

```diff
@@ -35,6 +36,7 @@ class SidePageComponent {
   constructor() {
     this.knowledgeGraphComponent = new KnowledgeGraphComponent();
     this.summaryKeyComponent = new SummaryKeyComponent();
+    this.selectionPopUpComponent = new SelectionPopUpComponent();
```

Comment:
Should this really be in SidePageComponent? I'm not 100% sure where I would put it, but maybe in readerview since html of this component is contained in html of reader view?